scalableminds/webknossos
# Improve import+formats docs and add explanations to upload UI

This replaces #5400. I had to re-open the PR, since I updated the docs and the preview only works when the branch is prefixed with `docs`.

For the docs, I reworked the structure and text quite a bit. The new structure is roughly as follows:
- Managing Datasets
  - explains high-level ways of getting data into webknossos (e.g., uploading, file system). for non-wkw formats, it refers to `Data Formats`
  - explains concepts, such as editing dataset settings and dataset sharing
- Data Formats
  - explains Concepts (Layers, Magnifications etc)
  - explains wk formats (wkw + nml)
  - explains tools
    - conversion with webknossos
    - conversion with wkcuber


### URL of deployed dev instance (used for testing):
- http://docsuploaduiformathint.webknossos.xyz/

### Steps to test:
- read https://docs.webknossos.org/v/docs-upload-ui-format-hint/guides/datasets
- read https://docs.webknossos.org/v/docs-upload-ui-format-hint/reference/data_formats
- go to http://docsuploaduiformathint.webknossos.xyz/datasets/upload and check out new explanation texts/popovers (only available when jobsEnabled=true)

![image](https://user-images.githubusercontent.com/2486553/115840459-0089c300-a41c-11eb-9c43-407489709211.png)


### Issues:
- fixes #5399

------
